### ISSUE TITLE
Possible things to do next (short term goals)

### ISSUE THREAD

**Issue by valeriupredoi** [valeriupredoi/bgc-val#5]
So now that the thing actually runs here are few things I'm thinking we should do in the short run:

**Definitely**

- improve that paths handling via (most probably) a yaml configuration file

**Probably good to have**

- making it a package and installing it centrally on JASMIN in e.g. `/apps/jasmin/community/`
- that would mean the tool is accessible via an executable that Andrew, Juju, Till etc can run, supplied with correct command line arguments - @ledm which is the executable best suited for this - `theWholePackage` perhaps?
- that would also need a decent configuration file that can be done from the point above about paths

**Comment by valeriupredoi** [comment]
Of course, I'd like to have a first release (official) here on GitHub before asking @agstephens to allow a deployment on JASMIN

**Comment by valeriupredoi** [comment]
OK I have now packaged it https://github.com/valeriupredoi/bgcval2

**Comment by ledm** [comment]
So, are we continuing this here, or moving these issues to https://github.com/valeriupredoi/bgcval2
